spec/services/atmosphere/optimizer_spec.rb

```diff
@@ -96,18 +98,18 @@
           appl_type.preference_memory = biggest_os_flavor.memory
           appl_type.save
 
-          tmpl, flavor = subject.select_tmpl_and_flavor([tmpl_at_amazon, tmpl_at_openstack])
+          tmpl, tenant, flavor = subject.select_tmpl_and_flavor_and_tenant([tmpl_at_amazon, tmpl_at_openstack])
```

Comment:
Useless assignment to variable - `tenant`. Use `_` or `_tenant` as a variable name to indicate that it won't be used.<br>Line is too long. [111/80]